ext/Shared/WeaponModification.lua

```diff
@@ -112,16 +112,18 @@ function WeaponModification:_ModifyWeapon(p_SoldierWeaponData, index, aimWorseni
 		return;
 	end
 
-	local botAimWorsening = aimWorseningNormal;
+	botAimRandomness = math.random (0, 2) / 10;
```

Comment:
I think we should wait with the aim-modification until its possible to do this for each weapon individually. I will have a look at it...